webservices/tasks/__init__.py

```diff
@@ -10,21 +11,37 @@
 schedule = {}
 if env.app.get('space_name', 'unknown-space').lower() != 'feature':
     schedule = {
+        # Refresh all Materialized views at 9am UTC daily
```

Comment:
Comments are usually good to have, but they suffer from the fact that they are often left unmodified when code changes. For this reason, it is often better to rename methods instead of adding comments. E.g., `webservices.tasks.refresh.refresh` should become `webservices.tasks.refresh.refresh_materialized_views`.

Instead of explaining that the schedule is for 9am UTC daily, it may be better to do the following:
1. Add a single comment highlighting that all schedules are UTC
2. Say _why_ 9am UTC is the best time for refreshing materialized views.